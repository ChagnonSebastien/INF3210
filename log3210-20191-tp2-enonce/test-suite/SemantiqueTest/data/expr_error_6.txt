Declaration
num a;
EndDeclaration
//number with boolean

a = -!1;
