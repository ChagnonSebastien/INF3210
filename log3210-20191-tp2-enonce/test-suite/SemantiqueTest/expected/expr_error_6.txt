Invalid type in expression got Number was expecting Bool